src/dotnet-install.ps1

```diff
@@ -434,6 +437,51 @@ function Get-LegacyDownload-Link([string]$AzureFeed, [string]$SpecificVersion, [
     return $PayloadURL
 }
 
+function Get-Product-Version([string]$AzureFeed, [string]$SpecificVersion) {
+    Say-Invocation $MyInvocation
+
+    if ($Runtime -eq "dotnet") {
+        $ProductVersionTxtURL = "$AzureFeed/Runtime/$SpecificVersion/productVersion.txt"
```

Comment:
minor:  Styling issue, should the conditions that result in the following ```$ProductVersionTxtURL = "$AzureFeed/Runtime/$SpecificVersion/productVersion.txt"``` be consolidated?

Reply:
I'm going to leave this for a future refactoring;  the property is already "obsolete" per the parameters at the top of the file and slated to be eventually removed (not sure how; will likely involve a new structure for the blobs on the storage account?), and I want to make as little change as possible to the existing codepaths.